mk/subdir.mk

```diff
@@ -90,36 +90,7 @@ $2: $$(depends-$1)
 	$(q)mkdir -p $4
 	$(q)$$(recipe-$2)
 
-cflags-$$(oname) 		:= $$(cflags-y) $$(cflags-$(1)-y)
-cflags-remove-$$(oname) 	:= $$(cflags-remove-y) \
-					$$(cflags-remove-$(1)-y)
-cxxflags-$$(oname) 		:= $$(cxxflags-y) $$(cxxflags-$(1)-y)
-cxxflags-remove-$$(oname) 	:= $$(cxxflags-remove-y) \
-					$$(cxxflags-remove-$(1)-y)
-cppflags-$$(oname) 		:= $$(cppflags-y) $$(cppflags-$(1)-y)
-cppflags-remove-$$(oname) 	:= $$(cppflags-remove-y) \
-					$$(cppflags-remove-$(1)-y)
-aflags-$$(oname) 		:= $$(aflags-y) $$(aflags-$(1)-y)
-aflags-remove-$$(oname) 	:= $$(aflags-remove-y) \
-					$$(aflags-remove-$(1)-y)
-incdirs-$$(oname)		:= $$(thissubdir-incdirs) $$(addprefix $(sub-dir)/,$$(incdirs-$(1)-y))
-# Clear local filename specific variables to avoid accidental reuse
-# in another subdirectory
-cflags-$(1)-y 			:=
-cflags-remove-$(1)-y		:=
-cflags-lib-y			:=
-cxxflags-$(1)-y 			:=
-cxxflags-remove-$(1)-y		:=
-cxxflags-lib-y			:=
-cppflags-$(1)-y			:=
-cppflags-remove-$(1)-y		:=
-cppflags-lib-y			:=
-aflags-$(1)-y 			:=
-aflags-remove-$(1)-y		:=
-incdirs-$(1)-y			:=
-fname				:=
-oname				:=
-
```

Comment:
"mk/subdir.mk: refactor process-subdir-{srcs-y,gensrcs-helper}":
That is probably OK but would be better as a separate commit

Reply:
What do you mean?

Reply:
Please ignore this comment. It was a mistake.